src/pages/Timeline/AddTimelineEntry/AddTimelineEntry.tsx

```diff
@@ -0,0 +1,96 @@
+import CategorySelect from '@components/EditTimeline/CategorySelect/CategorySelect';
+import EditDateSelect from '@components/EditTimeline/DateSelect/EditDateSelect';
+import EditTextInput from '@components/EditTimeline/EditTextInput/EditTextInput';
+import ProjectSelect from '@components/EditTimeline/ProjectSelect/ProjectSelect';
+import { useAppSelector, useAppDispatch } from '@redux/hooks';
+import { selectAuth } from '@redux/slices/AuthRedux';
+import useAxios from '@services/hooks/useAxios';
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import styles from './AddTimelineEntry.module.css';
+import { appActions } from '@redux/slices/AppRedux';
+
+interface TimelineProps { }
+
+interface TimelineInfo {
+  title: string,
+  description: string,
+  date: string,
+  project: string,
+  author: string,
+  categories: Array<string>,
+  contributors: string,
+}
+
+const AddTimelineEntry: React.FC<TimelineProps> = (props) => {
+  const { access_token } = useAppSelector(selectAuth);
+  const dispatch = useAppDispatch();
+
+  const [timeline, setTimeline] = useState<TimelineInfo>({title: "", description: "", date: "2004-01-01", project: "", author: "", categories: [], contributors: ""});
+
+  const history = useHistory();
+  
+  const add = async () => {
+    const splitParagraphs = (description: string) => {
+        return description.split('\n');
+    }
+
+    const snapshot = {
+      author: timeline.author,
+      title: timeline.title,
+      project: timeline.project,
+      date: timeline.date,
+      categories: timeline.categories, 
+      descriptions: splitParagraphs(timeline.description),
+      hyperlinks: ["google.com"],
+      contributors: timeline.contributors.split(",").map((c: string) => c.trim())
+    }
+    await axios({
+      method: "POST",
+      url: "http://localhost:4000/api/snapshots",
+      data: snapshot,
+      headers: {
+        authorization: access_token
+      }
+    })
+    .then(() => {
+        history.push("/timeline");
+        dispatch(appActions.setAlert("Add Entry Successful!"));
+    }).catch(err => {
+        dispatch(appActions.setAlert(err.message));
+    })
+  }
+
+  return (
+    <div className={styles.page}>
+      <div className={styles.container}>
```

Comment:
I suppose following the current pattern of giving classNames to each element in between quotation marks would be best for consistency's sake 🤔  So for example:

```suggestion
  return (
    <div className="add-snapshot-page">
      <div className="add-snapshot-container">
```

Additionally, for the styles that are shared with EditTimelineEntry, both elements can just use the same css styling.

Reply:
okay!